components/org.wso2.carbon.identity.data.publisher.audit.common/src/main/java/org/wso2/carbon/identity/data/publisher/audit/common/AuditDataPublisherUtils.java

```diff
@@ -22,30 +22,41 @@
 import org.wso2.carbon.base.MultitenantConstants;
 import org.wso2.carbon.identity.core.util.IdentityTenantUtil;
 
+/**
+ * Audit data publisher related utilities.
+ */
 public class AuditDataPublisherUtils {
+    private AuditDataPublisherUtils() {     // Prevent initializing
```

Comment:
Shall we add a new line between the class definition and the constructor?

Reply:
Added